### PR TITLE
Transform cfa info fetching HOC to hook

### PR DESCRIPTION
**Pull request by stephane-ruhlmann**

### CODE REVIEW

File: ui/src/common/hooks/useFetchCfaInfo.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { useQuery } from "react-query";
+
+import { fetchCfa } from "../api/tableauDeBord";
+
+const useFetchCfaInfo = (cfaUai) => {
+  const { data, isLoading, error } = useQuery(["cfas", cfaUai], () => fetchCfa(cfaUai));
+
+  return { data, loading: isLoading, error };
```
Comment (sbenfares) [inline ui/src/common/hooks/useFetchCfaInfo.js]: pkoi pas garder le isLoading

**Comment by stephane-ruhlmann** [comment]
@sbenfares A croire que tu lis dans mes pensées, j'ai créé ça hier : https://trello.com/c/ePqL4dlF/1801-transformers-les-hoc-en-hooks-quand-cest-pertinent 😄

**Comment by sbenfares** [comment]
> @sbenfares A croire que tu lis dans mes pensées, j'ai créé ça hier : https://trello.com/c/ePqL4dlF/1801-transformers-les-hoc-en-hooks-quand-cest-pertinent 😄

Lol, je lisais ça tout à l'heure : 

https://medium.com/javascript-scene/do-react-hooks-replace-higher-order-components-hocs-7ae4a08b7b58

-> You may have a poor use-case for HOCs if:
The behavior requires adding a bunch of props to a component.
The behavior is only used in one component.
The behavior must be customized for each component which uses it.

-> You may have a good use-case for HOCs if:
The behavior is not specific to any single component, but rather applies to many or all components in the app, and
The behavior doesn’t need to provide a bunch of props to the components that use it.
Components can be used stand-alone without the behavior from the HOC.
No custom logic needs to be added to a component being wrapped by the HOC.

File: ui/src/common/hooks/useFetchCfaInfo.js (hunk 1 above)
Comment (stephane-ruhlmann) [reply ui/src/common/hooks/useFetchCfaInfo.js]: J'avais voulu garder le même naming que le hook `useFetch` pour être cohérent à travers la codebase

**Pull request merged**